### ISSUE TITLE
ViewPoint Workstation Administrator desktop error.

### ISSUE THREAD

**Issue by Xuser641** [livingcomputermuseum/Darkstar#18]
I have a Workstation Aministrator Desktop on the emulated star but when I went to log in it said there were no Workstation Administrator desktops created and gave me the menu to create one. Choosing list local desktops showed that my desktop exists and that it is a Workstation Administrator desktop but it won't let me log in. If I create another Workstation Administration desktop and log out of it then the login menu will appear and it will allow me to log in to the original desktop. How do I fix this so that I don't have to temporarily create a new desktop just to login to the old one. 

Host system:
Dell Inspiron 545
Windows Vista 32bit
Pentium Dual Core E5200 @ 2.50GHz
3GB RAM
Darkstar Version 1.1.0.0

Emulated System: (Info from Loader)
Viewpoint 2.0.6 installed from the disks on bitsavers (VP Standalone)
VP Document Editor 2.0.5
Software Options Tool 2.0
Keyboard Accelerators 2.0
VP Xerox Classic Fonts 1.0
VP Xerox Modern Fonts 1.0
VP Data Driven Graphics 2.0

Also while getting the info required to write this I found out that Viewpoint seems to have randomly decided that my licence key for the VP Document Editor is invalid and that the required software option is not installed (Everything was working perfectly yesterday)

**Comment by Xuser641** [comment]
Is there anywhere I can get help on this?

**Comment by livingcomputermuseum** [comment]
How is your clock configured in the emulator?  If the date changes in certain ways, this can cause the Product Factoring (licensing) to expire, resulting in the behavior you're seeing.  If the date gets reset to before the time the system was installed and licensed, or is adjusted too far into the future (or has a post-y2k date applied) it can cause this behavior.  It's very important to keep the date/time consistent when working with Viewpoint.

**Comment by Xuser641** [comment]
The Date is set to 12/01/1997 (in US date format) which is the same date it was set to when the software was installed.

**Comment by bluefin88** [comment]
I am experiencing the same issue. I have emulator TOD set as 12/1/1997. After launching viewpoint from HD image, I am also required to create another administrator desktop. After creating an administrator desktop, I am then able to log in using user:xerox:star. 

However, I am also unable to successfully enabled Software Options following the instructions from 610E12440_Viewpoint_2.0_Software_Installation_Jul88.pdf and using the following password: ViewPoint 2.0 / Services 11.0: 8 7T78 M8YL LFEQ.

Basically every time I click 'Set Options', my selected software list would clear with nothing selected, even if I have the correct password inputted.

**Comment by techfury90** [comment]
I've run into this- the problem is that the 8 7T78 M8YL LFEQ key is sort of a backdoor. The usual flow as outlined in 610E12440 involves a different mechanism in which you input the key and just press Set Options, and the key defines what is enabled.

Since the process is completely undocumented otherwise, here is the flow: (note: I'm running from memory here, if something doesn't make sense, just reply and I'll get off my ass and fire up Darkstar to check)

1. Open up the Software Options dialog.
2. Enter the 8 7T78 M8YL LFEQ key and press Set Options. 
3. Select items to license. 
**IMPORTANT** You must press Save Options (or maybe it's Store Options? **NOT** Set Options) before proceeding to the next group of things to license. If you change which screen of licensable things is being shown, your selections **WILL** be lost if you do not click Save Options.
**ALSO IMPORTANT** You also cannot select VP Standalone, VP NetCom, and/or VP RemoteCom as these options are all mutually exclusive. Only choose VP Standalone unless you're building an XNS network client (server must be up beforehand), in which case it would be VP NetCom.
4. Exit the Software Options dialog.

**Comment by Xuser641** [comment]
I have tried what you said and it is working now (but I have only just done it so hopefully it won't break again). Has anyone been able to work out what is going on with the Star not letting me sign in to the desktop until I create a new one?
Also, just to confirm did you mean do not factor VP Standalone, NetCom or RemoteCon or that you can not select more than one. I expect it's the second one but just in case I though I should check.
